packages/terra-application/src/Application.scss

```diff
@@ -0,0 +1,23 @@
+@import '~terra-mixins';
```

Comment:
terra-application doesn't currently have a dependency on terra-mixins. Should we remove this?